Cargo.toml

```diff
@@ -174,7 +174,9 @@ tokio = "1.32.0"
 tracing = "0.1.37"
 tracing-attributes = "0.1.26"
 tracing-subscriber = "0.3.17"
-web-push = { version = "0.10.0", default-features = false, features = ["hyper-client"] }
+web-push = { version = "0.10.0", default-features = false, features = [
```

Comment:
Was this purposely changed? Every other dependency is on a single line

Reply:
Auto-format on save